libs/affinity/src/parse_affinity_options.cpp

```diff
@@ -13,8 +13,6 @@
 
 #include <hwloc.h>
 
-#include <boost/variant.hpp>
-
```

Comment:
Boost Spirit relies internally on `boost::variant` (for parsing alternatives). We can remove this `#include` here as we don't directly refer to the type. This does not mean however that this code doesn't depend on it anymore...